src/Rules/Functions/ImplodeFunctionRule.php

```diff
@@ -0,0 +1,64 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Functions;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Php\PhpVersion;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\Type\Accessory\AccessoryNonEmptyStringType;
+use PHPStan\Type\ErrorType;
+use PHPStan\Type\TypeUtils;
+use PHPStan\Type\UnionType;
+
+/**
+ * @implements \PHPStan\Rules\Rule<\PhpParser\Node\Expr\FuncCall>
+ */
+class ImplodeFunctionRule implements \PHPStan\Rules\Rule
+{
+	public function getNodeType(): string
+	{
+		return FuncCall::class;
+	}
+
+	public function processNode(Node $node, Scope $scope): array
+	{
+		if (!($node->name instanceof \PhpParser\Node\Name)) {
+			return [];
+		}
+
+		$name = strtolower((string) $node->name);
+		if ($name !== 'implode') {
+			return [];
+		}
+
+		$args = $node->getArgs();
+		if (count($args) !== 2) {
```

Comment:
You can also call `implode` with a single argument that should be checked here: https://www.php.net/manual/en/function.implode.php